collections/_article/dev-snapshot-godot-4-3-dev-1.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Dev snapshot: Godot 4.3 dev 1"
+excerpt: "We're ending the year with an early sneak peak at the upcoming Godot 4.3 release with its first dev snapshot!"
+categories: ["pre-release"]
+author: Yuri Sizov
+image: /storage/blog/covers/dev-snapshot-godot-4-3-dev-1.webp
+image_caption_title: "Yeetus"
+image_caption_description: "A game by YarnCat Games"
+date: 2023-12-21 17:00:00
+---
+
+This was an exciting year for Godot! Our contributors managed to deliver massive, feature-packed releases 3 times over our previous norm. We can't wait to see and share what the international community of passionate game and game engine developers can create for you in the year 2024. Actually, they have already started their work on the upcoming Godot 4.3 release.
+
+And it's a holiday season, so we cannot leave you without a small gift as we wrap up the production and plan our leisure time. Please, enjoy the first dev snapshot of Godot 4.3, which includes quite a few noteworthy improvements for you to play with. It also contains a handful of fixes, including pretty much everything [published last week in Godot 4.2.1](/article/maintenance-release-godot-4-2-1/).
```

Comment:
The end sentence seems a little odd. Why would a 4.3 release *not* contain fixes released in 4.2.1? I mean, why mention it?

Reply:
We mention it so people can read those release notes and learn about things we don't mention in these release notes.

Reply:
Ah OK. Maybe you could do something like:
```suggestion
And it's a holiday season, so we cannot leave you without a small gift as we wrap up the production and plan our leisure time. Hot on the heals of [last week's release of 4.2.1](/article/maintenance-release-godot-4-2-1/), please enjoy the first dev snapshot of Godot 4.3, which includes quite a few noteworthy improvements for you to play with, as well as some more fixes.
```

Reply:
I don't think this implies what I want it to imply. What feel wrong to you in the current structure of the statement?

Reply:
As I mentioned, it just felt odd to me saying 4.3 contains 4.2.1's fixes (although I note that you imply it's only most of them - have some been reverted, and if so, which?). But you know better than me what people might ask questions about, so if some people don't realise that 4.x contains fixes from 4.x-1.y, then sure, keep the existing text!

Reply:
Yeah, we get asked this from time to time, and have to clarify. In this instance, it's just a device to add a link and avoid repeating the same high profile fixes already listed in the previous article.

As for me saying "pretty much", it's not really important. I can say that it includes everything. There are a couple of commits in 4.2.1 which were made specifically for 4.2 and aren't relevant/addressed differently in 4.3. But it shouldn't matter to most readers. So if you think it's fine with these caveats, then I'd keep it as is 🙃 

Thanks for your thorough reviews, by the way, always! They are very appreciated, and don't let me ever make you feel like they are not.